deploy/playbook-single.yml

```diff
@@ -1,8 +1,7 @@
 - hosts: mainserver,executionnodes
   become: yes
   tasks:
-    - raw: which python || apt-get update
```

Comment:
- raw: apt-get update

above line would ensure that apt repo index is updated. Needed when installing on a fresh, clean OS with no prior updates.